api/swagger.yaml

```diff
@@ -3046,14 +3046,36 @@ definitions:
         type: "object"
         properties:
           Constraints:
-            description: "An array of constraints."
+            description: |
+              An array of constraint expressions to limit the set of nodes where
+              a task can be scheduled. Constraint expressions can either use a
+              _match_ (`==`) or _exclude_ (`!=`) rule. Multiple constraints find
+              nodes that satisfy every expression (AND match). Constraints can
+              match node or Docker Engine labels as follows:
+
+              node attribute       | matches                  | example
+              ---------------------|--------------------------|-----------------------------------------------
+              `node.id`            | Node ID                  | `node.id==2ivku8v2gvtg4`
+              `node.hostname`      | Node hostname            | `node.hostname!=node-2`
+              `node.role`          | Node role                | `node.role==manager`
```

Comment:
What about something like this in the `matches` column: `Node role (manager|worker)` ?

Reply:
Updated 👍